### ISSUE TITLE
Mark's RSE conf talk

### ISSUE THREAD

**Issue by maelle** [ropensci-org/monthly#247]
When the recording becomes available cc @mpadge

**Comment by maelle** [comment]
@mpadge any update on recordings from RSE conf?

**Comment by mpadge** [comment]
Nothing seems to have happened. Maybe they won't put them up at all?

**Comment by maelle** [comment]
in that case could you add a link to the slidedeck

- to the event page on roweb3 see example https://github.com/ropensci/roweb3/commit/eeeb7d3512b0760c95af2f30ebfce2c6a8a810fb
- as a short paragraph in the HQ section in the newsletter PR? https://github.com/ropensci/roweb3/pull/868

Thank you :pray:

**Comment by maelle** [comment]
@mpadge no news of a potential recording?

**Comment by mpadge** [comment]
Yes, finally published: https://www.youtube.com/watch?v=7H4FCpZa3w8

**Comment by maelle** [comment]
yay! adding it to HQ now.
